openedx_tagging/core/tagging/rest_api/paginators.py

```diff
@@ -0,0 +1,66 @@
+from edx_rest_framework_extensions.paginators import DefaultPagination  # type: ignore
+from rest_framework.response import Response
+
+# From this point, the tags begin to be paginated
+TAGS_THRESHOLD = 1000
+
+# From this point, search tags begin to be paginated
+SEARCH_TAGS_THRESHOLD = 200
+
+
+class TagsPagination(DefaultPagination):
+    """
+    Custom paginator that returns the range index of the current page
```

Comment:
Why is this useful/necessary? As far as I can see, we're not using it (yet).

It's always a good idea to put the "why" into the docstring so that in the future people know why it's here and can remove the code if it's not needed. If we don't say why and it never gets used, the code may sit around forever anyways, if nobody knows whether it's important or not.

Reply:
If it's for https://github.com/openedx/modular-learning/issues/111 , I'm actually going to suggest that we exclude that from the MVP or at least do it later as part of Step 4, not implement it now in Step 2.

Reply:
> It's always a good idea to put the "why" into the docstring so that in the future people know why it's here and can remove the code if it's not needed.

It is because of [this discussion](https://www.figma.com/file/DmUC2e7BAoQsGDq4KgApPa/Content-Tagging-MVP?type=design&node-id=0-1&mode=design&t=cagvr8rMST58RI3N-0#521112066). The range index of the current page is [shown in the UI](https://www.figma.com/file/DmUC2e7BAoQsGDq4KgApPa/Content-Tagging-MVP?type=design&node-id=9074-106473&mode=design&t=VwsZIXCkpUqHBuXO-4)

Reply:
> If it's for https://github.com/openedx/modular-learning/issues/111 , I'm actually going to suggest that we exclude that from the MVP or at least do it later as part of Step 4, not implement it now in Step 2.

Ok, I see. In that case, should I remove everything related to range index?

Reply:
I updated this here: https://github.com/openedx/openedx-learning/pull/78/commits/6dded73c3e264d2126f9309de8a9fcada6577877

Reply:
Yeah, that's more of a UI thing. Especially when we're tight for budget, let's exclude it for now and we can consider adding it later when we implement that story. Also I think it makes more sense to implement it generally for the whole platform, not just for the tagging use case.

Reply:
> Also I think it makes more sense to implement it generally for the whole platform, not just for the tagging use case.

I have [implemented](https://github.com/openedx/openedx-learning/commit/6dded73c3e264d2126f9309de8a9fcada6577877) it that way because of how special this tag view is. We have to change the paginator to disable pagination in that view.

Reply:
Ah ok. Well if you want to keep what you have it's fine with me. I just don't want to spend any more time on it now.